include/machine/riscv/sifive_u/sifive_u_traits.h

```diff
@@ -24,7 +25,8 @@ template<> struct Traits<Machine>: public Traits<Machine_Common>
     static const bool supervisor = !library;                                                    // Run EPOS library in machine mode (works in supervisor as well)
 
     // CPU numbering
-    static const unsigned long CPU_OFFSET       = supervisor ? 1 : 0;                           // We skip core zero, which is a E CPU without MMU
+    static const unsigned long CPU_OFFSET       = (multicore || supervisor) ? 1 : 0;                           // We skip core zero, which is a E CPU without MMU
+    static const unsigned int  BSP              = 0;                                            // Bootstrap/service processor (not counting heterogeneos processor)                         // We skip core zero, which is a E CPU without MMU
```

Comment:
```suggestion
    static const unsigned int  BSP              = 1;                                            // Bootstrap/service processor (not counting heterogeneos processor)                         // We skip core zero, which is a E CPU without MMU
```

Alterei pra comecar em 1 e o bug da mmu sumiu e o deadlock inicial tbm o bug descrito no comentario do arquivo rv64_cpu_init.cc